### PR TITLE
Create associated class

### PR DESCRIPTION
**Pull request by opxz7148**
~# DONT MERGE THIS BRANCH YET~
~It ok to merge right now~
Note: 
- I have create a new model Attend to store which user attend each activity.
- I have revise create activity accord to new models.
- I have revise join to but right now user can still able to join activity that they've join already so this must be fix.
- I've revise some of test but not all. Test file with _ in a front of a file name is a one that I'm not yet revise.
- I've shortcut function but look like it more verbose should fix it's later.
## Don't forget to re-migrate before continue a new work

### CODE REVIEW

**Comment by opxz7148** [comment]
Already fetch code from main. I've edit @Thanawas-Sirilertsathit  code on edit activity handler function. From now on user can't edit number of people anymore accord to new models. From front end view there're no need to change since edit activity just simply ignore people value from post req.

**Comment by OmegaOoh** [comment]
> Already fetch code from main. I've edit @Thanawas-Sirilertsathit code on edit activity handler function. From now on user can't edit number of people anymore accord to new models. From front end view there're no need to change since edit activity just simply ignore people value from post req.

You should says "its no need to change the API usage." instead of "From front end view there're no need to change" as it can be misunderstand.

File: activities/tests/test_index.py
Comment (OmegaOoh) [inline activities/tests/test_index.py:1]: Please explain the reason for commenting test cases

**Review by OmegaOoh — CHANGES_REQUESTED** [review]
If the response from create activity won't be used/ is reassigned by another response without usage, assign it to `_` or just simply remove it from return of the function, which I don't see the use case of it at the moment

**Comment by Thanawas-Sirilertsathit** [comment]
I didn't login yet but pressed join and it joined. After login it still said I already joined.

File: activities/tests/test_index.py
Comment (opxz7148) [reply activities/tests/test_index.py:1]: I'm just forget to uncomment It. Already fix.

**Comment by opxz7148** [comment]
> If the response from create activity won't be used/ is reassigned by another response without usage, assign it to `_` or just simply remove it from return of the function, which I don't see the use case of it at the moment

Respond from create activity still necessary in some test case. Mostly in create activity test.

File: activities/tests/test_detail.py
Comment (OmegaOoh) [inline activities/tests/test_detail.py:1]: response from create_activity has not been used and reassigned immediately after the initial assignment

File: activities/tests/test_join.py
Comment (OmegaOoh) [inline activities/tests/test_join.py:1]: This file also contains unused response from create_activity

File: activities/tests/test_index.py
Comment (OmegaOoh) [inline activities/tests/test_index.py:1]: This file contains multiple unused response from create_activity

File: activities/tests/test_join.py [hunk 1]
```diff
@@ -1,24 +1,80 @@
...
+        self.assertEqual(response.status_code, 401)
+
     def test_join(self):
         """Join will increase number of people in activity."""
-        activity = create_activity("Joinable", 2, 1)
-        response = self.client.post(urls.reverse("activities:join", args=[activity.id]))
-        activity.refresh_from_db()
-        self.assertEqual(activity.people, 2)
-        self.assertJSONEqual(response.content, {"message": f"You successfully joined {activity.name}"})
+        response, new_act = create_activity()
```
Comment (OmegaOoh) [inline activities/tests/test_join.py:34]: This response is redundant

**Review by OmegaOoh — CHANGES_REQUESTED** [review]

**Comment by OmegaOoh** [comment]
> > If the response from create activity won't be used/ is reassigned by another response without usage, assign it to `_` or just simply remove it from return of the function, which I don't see the use case of it at the moment
> 
> Respond from create activity still necessary in some test case. Mostly in create activity test.

Don't make me comment every other one of them

**Review by OmegaOoh — APPROVED** [review]
Good job, Go ahead and merge it

**Pull request merged**